core/tee/tee_rpmb_fs.c

```diff
@@ -638,13 +625,11 @@ static TEE_Result tee_rpmb_req_pack(struct rpmb_req *req,
 			       RPMB_NONCE_SIZE);
 
 		if (rawdata->data) {
-#ifdef CFG_ENC_FS
 			if (fek)
 				encrypt_block(datafrm[i].data,
 					rawdata->data + (i * RPMB_DATA_SIZE),
 					*rawdata->blk_idx + i, fek);
 			else
```

Comment:
It's certainly used. If it's an issue or not is in my opinion not in scope of this P-R.